### ISSUE TITLE
Export  failure of TensorRT 10.11 when running scaled dot product on GPU A6000

### ISSUE THREAD

**Issue by evolvingai** [NVIDIA/TensorRT#4482]
I had an export issue for attention layer in a video transformer architecture  in tensorrt 10.7 and seemed to be fixed in 10.11. However when I exported using 10.11 on a windows machine the problem re-appeared. I noticed there was another issue #4333. 

 def _multi_head_self_attn(self, x):
        """
        x: [B, S, d_model]
        returns: [B, S, d_model]
        """
        B, S, D = x.shape
        qkv = self.qkv_proj(x)                         # [B, S, 3*D]
        qkv = qkv.view(B, S, 3, self.num_heads, self.d_head)
        q, k, v = qkv.unbind(dim=2)                    # each [B, S, H, d_head]
        # transpose to [B, H, S, d_head]
        q, k, v = [t.permute(0, 2, 1, 3) for t in (q, k, v)]

        # scaled dot-prod: [B, H, S, S]
        scores = torch.matmul(q, k.transpose(-2, -1)) / math.sqrt(self.d_head)
        weights = F.softmax(scores, dim=-1)
        attn   = torch.matmul(weights, v)              # [B, H, S, d_head]
        attn   = attn.permute(0, 2, 1, 3).contiguous()
        attn   = attn.view(B, S, D)                    # [B, S, d_model]
        return self.out_proj(attn)

ONNX               : [[ 4.624525  -1.0358317 -6.621598 ]], [[9.9651778e-01 3.4691526e-03 1.3012363e-05]],  argmax=[0]
ONNX               : 218.00 ms per video (n=54)
Input shape video_np: (1, 54, 3, 384, 384)
Input shape positions_np: (1, 54)
TRT                : [[-0.6519211   0.93750143 -1.7938591 ]], [[0.16076867 0.78791404 0.05131733]], argmax=[1]
TRT                : [[ 3.5375547 -2.3099315 -8.087528 ]], [[9.9711227e-01 2.8788093e-03 8.9132209e-06]],  argmax=[0]
Speedup (ONNX vs pytorch) : 0.97×
Speedup (TRT vs ONNX)     : 2.16×
Speedup (TRT vs PyTorch): 2.10×

 TensorRT Version: 10.11

NVIDIA GPU: A6000

NVIDIA Driver Version: Driver Version: 536.25

CUDA Version: Cuda compilation tools, release 12.2, V12.2.91
Build cuda_12.2.r12.2/compiler.32965470_0

CUDNN Version:

Operating System: Windows

Python Version (if applicable): 3.10

PyTorch Version (if applicable): 2.5.0

**Comment by LeoZDong** [comment]
Could you attach the error log and a repro script? Is the error on exporting torch -> ONNX, or running the exported ONNX model in TRT?
